src/templates/bbb-enter-name.de.html

```diff
@@ -0,0 +1,14 @@
+<h1>Dem Raum „{{room_name}}“ beitreten</h1>
+
+<div class="form-group">
+    <form method="get">
+        <div class="col30"><strong><label for="name">Dein Name</label></strong></div>
+        <div class="col70">
+            <input type="text" id="name" name="name" class="form-element" maxlength="50" aria-describedby="name-help">
+            <p id="name-help" class="form-element-help">Den Namen, den Du hier angibst, sehen alle Teilnehmenden. Du kannst natürlich auch ein Pseudonym verwenden. Gib gerne auch Deine Pronomen und ggf. Organisation in Klammern an.</p>
+        </div>
+        <div class="clearfix" style="margin-bottom: 20px"></div>
+
+        <input type="submit" class="button button-primary" id="submitButton" style="float: right;" value="Beitreten">
```

Comment:
Shouldn‘t we tell users here that they will be sent to a thrid party provider?

Reply:
Fair point. It was late at night already on my "day off" and I just wanted to get this done. :D